Core/include/Acts/EventData/detail/DynamicColumn.hpp

```diff
@@ -68,14 +84,26 @@ struct DynamicColumn<bool> : public DynamicColumnBase {
   }
 
   void add() override { m_vector.emplace_back(); }
+  void reserve(size_t size) override { m_vector.reserve(size); }
   void clear() override { m_vector.clear(); }
   void erase(size_t i) override { m_vector.erase(m_vector.begin() + i); }
   size_t size() const override { return m_vector.size(); }
 
-  std::unique_ptr<DynamicColumnBase> clone() const override {
+  std::unique_ptr<DynamicColumnBase> clone(bool empty) const override {
```

Comment:
`clone()` also clones the contents. Basically I need a way to make a new empty column with the same type.

Reply:
By point is that the specialisation for boolean type looks very identical to generic `clone`. I may have overlooked the subtle difference though.

Reply:
The `bool` specialization is exactly identical to the generic one. The only reason I'm specializing this is that I can't use the generic one, because in turn `std::vector<bool>` is specialized, where it's not possible to get a reference to the elements.